test/arns.test.js

```diff
@@ -155,36 +184,64 @@ describe('ArNS', async () => {
   });
 
   it('should increase the undernames', async () => {
-    const buyUndernameResult = await handle({
-      Tags: [
-        { name: 'Action', value: 'Buy-Record' },
-        { name: 'Name', value: 'test-name' },
-        { name: 'Purchase-Type', value: 'lease' },
-        { name: 'Years', value: '1' },
-        { name: 'Process-Id', value: ''.padEnd(43, 'a') },
-      ],
-    });
-    const increaseUndernameResult = await handle(
-      {
-        Tags: [
-          { name: 'Action', value: 'Increase-Undername-Limit' },
-          { name: 'Name', value: 'test-name' },
-          { name: 'Quantity', value: '1' },
-        ],
-      },
-      buyUndernameResult.Memory,
-    );
-    const result = await handle(
-      {
-        Tags: [
-          { name: 'Action', value: 'Record' },
-          { name: 'Name', value: 'test-name' },
-        ],
-      },
-      increaseUndernameResult.Memory,
-    );
-    const record = JSON.parse(result.Messages[0].Data);
-    assert.equal(record.undernameLimit, 11);
+    const runIncreaseUndername = async (sender) => {
```

Comment:
nit: consider changing to `assertIncreaseUndername` given these function contains the assertions